src/bin/bindgen.rs

```diff
@@ -294,14 +66,20 @@ pub fn main() {
         }
     }
 
-    let (options, out) = parse_args_or_exit(bind_args);
```

Comment:
This would remain something like:

```rust
let (options, out) = match bindgen::builder_from_flags(env::args()) {
    Ok((builder, Some(out)) => (builder, out),
    _ => // Exit, I guess.
};
```